Sources/Rendering/OpenGL/glsl/vtkVolumeFS.glsl

```diff
@@ -1314,6 +1346,63 @@ bool valueWithinScalarRange(vec4 val, vec4 min, vec4 max) {
   return withinRange;
 }
 
+#if vtkBlendMode == 22 &&  defined(UseIndependentComponents)
```

Comment:
This mode fails with the ColorMixPreset `ColorMixPreset.CUSTOM`
Is it possible to make it work with custom shader code?

Reply:
> actor.getProperty().setIndependentComponents(false)

I think that is the expected behavior. This new blend mode makes use of the independent components, so it doesn't work without them. It encodes the label map in a separate component.


> This mode fails with the ColorMixPreset ColorMixPreset.CUSTOM

Unfortunately, I don't have the capacity to check this one. Would it be possible to merge this and come back to it later?

Reply:
As in my new comment below, you should make sure that `UseIndependentComponents` is defined when you are in this mode: in `publicAPI.useIndependentComponents` of the opengl volume mapper, add the check `model.renderable.getBlendMode() === BlendMode.LABELMAP_EDGE_PROJECTION_BLEND`

Reply:
Done